src/sema/ast.rs

```diff
@@ -24,6 +24,7 @@ use std::{
 };
 use tiny_keccak::{Hasher, Keccak};
 
+#[cfg_attr(test, derive(Default))]
```

Comment:
Now that you are not using the default for `Type` anymore, is this attribute still necessary?

Reply:
No. I must have forgotten to `git push` before I left yesterday, sorry